internal/handlers/sqlite/msg_explain.go

```diff
@@ -83,6 +83,32 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		qp.Filter = params.Filter
 	}
 
+	// Skip sorting if there are more than one sort parameters
+	if h.EnableSortPushdown && params.Sort.Len() == 1 {
+		iter := params.Sort.Iterator()
```

Comment:
Let's not use iterator if we don't iterate

Reply:
What do we do if we want to just get the single key:value? Should we use params.Sort.Keys()[0] and params.Sort.Values()[0] instead?

Reply:
https://github.com/FerretDB/FerretDB/pull/3504#discussion_r1360159515